### ISSUE TITLE
Cleanup Jenkins agent image labels

### ISSUE THREAD

**Issue by michaelsauter** [opendevstack/ods-quickstarters#506]
We use the labels as specified by RedHat. I think they are only useful for their build service and make no sense for us at all. I think we should just remove them.

While we are at it, we could remove the `maintainer` label too.

Thoughts @faust2199 @gerardcl @henrjk @oalyman?

**Comment by gerardcl** [comment]
Agree! cleanup is always welcome 👍 I think the `maintainer` removal has already been done at some places indeed..

**Comment by henrjk** [comment]
@michaelsauter: Do you mean for example the label defined at https://github.com/opendevstack/ods-quickstarters/blob/883607ea1f2e3fc27e24d36d450b313cc4554f8f/common/jenkins-agents/golang/ocp-config/bc.yml#L25

**Comment by michaelsauter** [comment]
Sorry for the confusion, I meant the image labels in the `Dockerfile`, like https://github.com/opendevstack/ods-quickstarters/blob/master/common/jenkins-agents/maven/docker/Dockerfile.ubi8#L3-L10.

**Comment by henrjk** [comment]
Thanks for the clarification. I am for removing Labels only useful to the RedHat builder. 

To me it appeared that `architecture` and `name` might be useful to keep. 

However I looked what standard labels might be defined and ended up at https://github.com/opencontainers/image-spec/blob/master/annotations.md#pre-defined-annotation-keys
I assumed that `LABEL` statement would become an annotation as per the OCI image format.

As architecture is not mentioned anywhere in the annotations it may belong more into the realm of the runtime-spec (https://github.com/opencontainers/runtime-spec).

Any thoughts about platform in particular?

**Comment by michaelsauter** [comment]
Hmm, no particular thoughts around platform.

I think the way I stated it originally was probably too drastic. We can keep `architecture` and `name`. I also found https://bugzilla.redhat.com/show_bug.cgi?id=1334605, which unfortunately leads to a 404, but the `io.k8s` labels could be useful too.

Maybe we start by unifying the labels and remove all `com.redhat` ones?
